src/integration/integration_suite_test.go

```diff
@@ -200,6 +201,18 @@ func stopApiServer() {
 	ginkgomon.Interrupt(processMap[APIServer], 5*time.Second)
 }
 
+func sendSigusr2Signal(component string) {
+	process := processMap[component]
+	if process != nil {
+		process.Signal(syscall.SIGUSR2)
+		Eventually(process.Wait(), 5*time.Second).Should(Receive(), "sent SIGUSR2 signal to process, failed to exit in time")
```

Comment:
Just send the signal, don't wait since it is test dependent.
The check on the process should occur in the tests. It is something I was going to mention since it is confusing for it to be hidden here.